### DISCUSSION TITLE
Playlist

### DISCUSSION THREAD

**Discussion by Prabhakarsinghrx** [KRTirtho/spotube#1072] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Pain

is it possible to add playlist option in this application , and  like if we add and song in x playlist in spotube it automatically add this playlist in spotify Vice Versa..

### Suggested solution

if you add this feature that will very help-full so we can completely replace spotify

**Comment by KRTirtho** [comment]
This is already available 

![image](https://github.com/KRTirtho/spotube/assets/61944859/ef736414-2961-460c-ba0d-17c06ae3df71)
![image](https://github.com/KRTirtho/spotube/assets/61944859/862f20af-358a-48ed-8cdf-d31b4beb17cb)

**Comment by KRTirtho** [comment]
If you add any song to any playlist in Spotube it'll automatically sync with Spotify.
Spotube uses the official Spotify API. It's a client for Spotify after all 😉
